tests/jballerina-semtype-test/src/test/resources/test-src/anydata2.bal

```diff
@@ -0,0 +1,3 @@
+// TB<:A
```

Comment:
Shall we add some doc comments in Java Test class what these signs mean?

Reply:
Added